pfcwd/main.py

```diff
@@ -157,31 +157,36 @@ def collect_config(self, ports):
                 self.multi_asic.display_option
             )
 
+        ports_found = False
         for port in ports:
             config_list = []
             config_entry = self.config_db.get_entry(
                 CONFIG_DB_PFC_WD_TABLE_NAME, port
             )
             if config_entry is None or config_entry == {}:
                 continue
+            ports_found = True
             for config in CONFIG_DESCRIPTION:
                 line = config_entry.get(config[1], config[2])
                 config_list.append(line)
             table.append([port] + config_list)
 
+        if not ports_found:
```

Comment:
Do we need a new var? Wouldn't checking 'table' empty suffice?

Reply:
table may not be empty if multiple ports on different ASICs are specified.